platform/nodejs/tests/snfe.js

```diff
@@ -103,4 +105,40 @@ describe('SNFE', () => {
             testSNFE(engine);
         });
     });
+
+    describe('Filter loading', () => {
+        beforeEach(async () => {
+            // This is in lieu of a constructor for a non-singleton.
+            await engine.useLists([]);
+        });
+
+        it('should not reject on no lists', async () => {
```

Comment:
Here we add tests for what we consider _specified_ behavior. For example, I have not added a test for `engine.useLists(null)`. In my mind I consider this unspecified. Would you like to specify what happens if `null` is passed, or would you rather leave it as unspecified? If it's specified then there must be a test for it right over here. If the test breaks, the component has broken its "contract" with the rest of the software world.